platform/ios/ios.xcodeproj/project.pbxproj

```diff
@@ -3781,6 +3781,7 @@
 				INFOPLIST_FILE = framework/Info.plist;
 				INSTALL_PATH = "$(LOCAL_LIBRARY_DIR)/Frameworks";
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
+				LLVM_LTO = YES_THIN;
```

Comment:
This enables LTO for the `dynamic` target — you’ll also want to do this for the `static` target.